player.py

```diff
@@ -1022,9 +1033,18 @@ def faerie_fire(self):
         self.gcd = 1.0
         self.omen_proc = True
         self.faerie_fire_cd = 6.0
-        self.dmg_breakdown['Faerie Fire (Feral)']['casts'] += 1
-
-        if self.log:
-            self.gen_log('Faerie Fire (Feral)', '', False, False, False)
 
-        return 0.0
+        if self.cat_form:
+            self.dmg_breakdown['Faerie Fire (Cat)']['casts'] += 1
+            if self.log:
+                self.gen_log('Faerie Fire (Cat)', '', False, False, False)
+            return 0.0
+
+        self.dmg_breakdown['Faerie Fire (Bear)']['casts'] += 1
+        miss = (np.random.rand() < self.spell_miss_chance)
+        damage_on_hit = self.faerie_fire_hit
+        self.gen_log('Faerie Fire (Bear)', damage_on_hit, miss, False, False)
```

Comment:
Should wrap this in an `if self.log:` block for consistency with the rest of the code.